### ISSUE TITLE
Heading elements (h1..h6) should be used

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#209]
Heading elements (h1..h6) should be used for structuring information on the page.&nbsp;Page has multiple H1s and then jumps to H4 then back to H1.

**Comment by philipashlock** [comment]
In general, this should already be addressed, but any specific places where this is an accessibility concern can be ticketed separately
